src/sentry/static/sentry/app/components/sidebar/broadcasts.jsx

```diff
@@ -124,12 +123,13 @@ const Broadcasts = createReactClass({
     let unseenBroadcastIds = this.getUnseenIds();
     if (unseenBroadcastIds.length === 0) return;
 
-    broadcastActions.markAsSeen(this.api, unseenBroadcastIds).then(data => {
-      this.setState({
-        broadcasts: this.state.broadcasts.map(item => {
+    markBroadcastsAsSeen(this.api, unseenBroadcastIds).then(data => {
+      this.setState(state => {
```

Comment:
if you're trying to have a similar style as the previous `setState`, you can use parens around the return object:

```
this.setState(state => ({
  broadcasts: state.broadcasts.map(item => {
    item.hasSeen = true;
    return item;
  })
}))
```

Reply:
Thanks!